fedimint-server/src/consensus/mod.rs

```diff
@@ -766,6 +766,52 @@ impl FedimintConsensus {
         Ok(())
     }
 
+    async fn accepted_transaction_status(
+        &self,
+        txid: TransactionId,
+        accepted: AcceptedTransaction,
+        dbtx: &mut DatabaseTransaction<'_>,
+    ) -> TransactionStatus {
+        let mut outputs = Vec::new();
+        for (out_idx, output) in accepted.transaction.outputs.iter().enumerate() {
+            let outpoint = OutPoint {
+                txid,
+                out_idx: out_idx as u64,
+            };
+            let outcome = self
+                .modules
+                .get_expect(output.module_instance_id())
+                .output_status(
+                    &mut dbtx.with_module_prefix(output.module_instance_id()),
+                    outpoint,
+                    output.module_instance_id(),
+                )
+                .await
+                .expect("the transaction was processed, so should be known");
```

Comment:
let's be exact here: as the transaction was processed, the output *must* be known. Or are there cases where the outcome is not known when a transaction is processed?